data/json/bionics.json

```diff
@@ -364,15 +358,15 @@
     "type": "bionic",
     "name": { "str": "Directional EMP" },
     "description": "Surgically mounted in the palm of your hand are small parabolic EMP field generators.  You may use power to fire a wide, but short-ranged blast which will disable electronics and robots.",
-    "occupied_bodyparts": [ [ "arm_r", 1 ], [ "hand_r", 2 ] ],
+    "occupied_bodyparts": [ [ "arm_l", 6 ], [ "hand_l", 1 ] ],
```

Comment:
why move it from right to left?

Reply:
Many CBMs, particularly weapons use the right hand, probably because it's a dominant one for most people.
I moved many utility ones to the left one.

Reply:
It would be nice to choose. Perhaps, until then, ones not requiring much aim/dexterity could be put in the left (e.g., a shotgun), and others in the right?